packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts

```diff
@@ -27,13 +27,15 @@ export function uploadFile({
       progressCallback,
       errorCallback,
       completeCallback,
+      contentType: file.type || 'binary/octet-stream',
       ...rest,
     });
   } else {
     return Storage.put(fileName, file, {
       level,
       resumable: false,
       progressCallback,
+      contentType: file.type || 'binary/octet-stream',
```

Comment:
nit: can we make a const of `file.type || 'binary/octet-stream'` so we don't have to repeat it?

Reply:
+1. Also would it make sense to use nullish coalescing here rather than the `OR` operator since the resolved value is not a boolean?